### PR TITLE
OSDOCS-8517-CCO Cloud Credential Operator bug text for 4.15 RNs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15

Issue: [OSDOCS-8517](https://issues.redhat.com/browse/OSDOCS-8517)

Link to docs preview: [Scroll to Cloud Credential Operator under Bug Fixes](https://71779--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-bug-fixes)

QE review:
Not needed

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1191,6 +1191,18 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
...
+
+* Previously, buckets created by running the `ccoctl azure create` command were prohibited from allowing public blob access due to a change in the default behavior of Microsoft Azure buckets. With this release, buckets created by running the `ccoctl azure create` command are explicitly set to allow public blob access. (link:https://issues.redhat.com/browse/OCPBUGS-22369[*OCPBUGS-22369*])
+
+* Previously, an Azure Managed Identity role was omitted from the Cloud Controller Manager service account. As a result, the Cloud Controller Manager could not manage service type load balancers in environments deployed to existing vnets with a private publishing method. With this release, the missing role was added to the Cloud Credential Operator utility (`ccoctl`) and Azure Managed Identity installations into an existing vnet with private publishing is possible. (link:https://issues.redhat.com/browse/OCPBUGS-21745[*OCPBUGS-21745*])
```
Comment (kcarmichael08) [inline release_notes/ocp-4-15-release-notes.adoc:1200]:
```suggestion
* Previously, an Azure Managed Identity role was omitted from the Cloud Controller Manager service account. As a result, the Cloud Controller Manager could not manage service type load balancers in environments deployed to existing VNets with a private publishing method. With this release, the missing role was added to the Cloud Credential Operator utility (`ccoctl`) and Azure Managed Identity installations into an existing VNet with private publishing is possible. (link:https://issues.redhat.com/browse/OCPBUGS-21745[*OCPBUGS-21745*])
```
It looks like "VNet" is correct based on the [existing docs](https://docs.openshift.com/container-platform/4.14/installing/installing_azure/installing-azure-vnet.html).

**Review by kcarmichael08 — COMMENTED** [review]
One comment; otherwise, LGTM!

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc]: Changed, thank you!

**Pull request merged**